runtime/signals_nat.c

```diff
@@ -234,15 +234,16 @@ DECLARE_SIGNAL_HANDLER(segv_handler)
        handler, we jump to the asm function [caml_stack_overflow]
        (from $ARCH.S). */
 #ifdef CONTEXT_PC
+    CONTEXT_C_ARG_1 = (context_reg) Caml_state;
```

Comment:
Shouldn't you change the declaration of `caml_stack_overflow` at line 210 of this file?

Reply:
Fixed.